### DISCUSSION TITLE
Nob question stt-model-manager

### DISCUSSION THREAD

**Discussion by zdgdg** [coqui-ai/TTS#2286] [General]
I don't understand programming. According to the tutorial from @thorstenmueller and his video, I launched TTS on Windows.

I need to run TTS with Czech.  **tts-server.exe --model_name tts_models/cs/cv/vits** has a very strange accent

I will download using **STT-Model-Manager** Czech STT v0.2.0 (Czech STT v0.3.0 does not work) to .local  files.

Is there any simple procedure what to do next?

**Comment by thorstenMueller** [comment]
Hi and thanks for watching my videos 😊.
It seems like you're mixing up two different aspects. What do you try to make?
1. STT (Speech-to-Text)
2. TTS (Text-to-Speech, which my videos are mostly about)

**Comment by zdgdg** [comment]
Hi, I try to TTS (text-to-Speech), but I didn't look much. Mr. Thorsten Müller are your videos Eigene Stimme klonen (1) - Einleitung etc?

**Reply by thorstenMueller** [reply]
Yes, that's my channel 😁.
https://www.youtube.com/@ThorstenMueller

**Reply by zdgdg** [reply]
Thank you for the answers and for the videos.
